terraform/services/backup-plan/main.tf

```diff
@@ -0,0 +1,79 @@
+module "standards" {
+  source      = "github.com/CMSgov/cdap//terraform/modules/standards?ref=0bd3eeae6b03cc8883b7dbdee5f04deb33468260"
+  app         = "cdap"
+  env         = var.env
+  root_module = "https://github.com/CMSgov/cdap/tree/main/terraform/services/backup-plan"
+  service     = "backup-plan"
+}
+
+data "aws_backup_vault" "primary" {
+  name = "CMS_OIT_Backups_Vault"
+}
+
+
+data "aws_backup_vault" "secondary" {
+  name     = "CMS_OIT_Backups_Vault"
+  provider = aws.secondary
+}
+
+resource "aws_backup_plan" "this" {
+  name = "4Hours1CA_Daily7_Weekly35_Monthly90"
+  #only the 4hr rule should be copied to secondary
+  rule {
+    rule_name         = "4Hours1CA"
```

Comment:
I've just realized the `CA` I suggested for this rule is not appropriate as this does not copy "Cross Account" but cross-region. A more fitting name would be `4Hours1DR` to denote "Disaster Recovery". This should be changed here, in the backup plan name, and in the selection tag.